atlas/query_node_API_sketch.graphql

```diff
@@ -0,0 +1,196 @@
+
+enum Language {
+  Chinese
+  English
+  Arabic
+  Portugese
+  French
+  # ...
+}
+
+type Channel {
+
+  id: ID!
+
+  # Id of underlying entity.
+  entityID: BigInteger!
```

Comment:
`id` is the instance identifier for `VideoMedia` as a concept in the query node, while `entityID` is the identifier used for the same concept as it exists in the underlying on-chain data model (the content directory). They are distinct. If you want to directly look something up in the content directory, e.g. talking to a full node, requires the latter.

Reply:
> `id` is the instance identifier for `VideoMedia` as a concept in the query node

Does that mean that this ID will be different for each query node instance?

Reply:
Yes in principle it could be, but in practice, I would not be surprised if it was not. But for safety, if you need to mix responses across nodes, it would be better to use the underlying entityID. If this creates nastiness, then we should look more closely at whether the guarantees about the `id` are stronger than I am assuming.